.github/workflows/ci.yaml

```diff
@@ -64,22 +52,30 @@ jobs:
         - uses: ./.github/actions/setup-env
           with:
             python-version: "3.11"
-        - name: Cache conda env
```

Comment:
Ignoring the caching issues I ran into with the integration build, I briefly looked at the past runs and found that this caching phase adds more time. So I've removed all CI conda caching in this project.